Casks/phpwebstudy.rb

```diff
@@ -0,0 +1,24 @@
+cask "phpwebstudy" do
+  version "1.0.14"
+  sha256 "e7d4ef8a485e69513ac0f860be013dd3405c0bcb2d474aa3bdeba4caf4257f05"
+
+  url "https://github.com/xpf0000/PhpWebStudy/releases/download/v#{version}/PhpWebStudy-#{version}-mac.zip", verified: "github.com/xpf0000/PhpWebStudy/"
+  name "phpwebstudy"
```

Comment:
Looking at the website the name should likely be this, as it should be stylised the same as in the upstream application.
```suggestion
  name "PhpWebStudy"
```